satellite/payments/stripecoinpayments/creditcards.go

```diff
@@ -82,3 +82,26 @@ func (creditCards *creditCards) Add(ctx context.Context, userID uuid.UUID, cardT
 
 	return nil
 }
+
+// MakeDefault makes a credit card default payment method.
+// this credit card should be attached to account before make it default.
+func (creditCards *creditCards) MakeDefault(ctx context.Context, userID uuid.UUID, cardID []byte) (err error) {
+	defer mon.Task()(&ctx, userID, cardID)(&err)
+
+	customerID, err := creditCards.service.customers.GetCustomerID(ctx, userID)
+	if err != nil {
+		return payments.ErrAccountNotSetup.Wrap(err)
+	}
+
+	params := &stripe.CustomerParams{
+		Params:        stripe.Params{},
+		DefaultSource: stripe.String(string(cardID)),
+	}
+
+	_, err = creditCards.service.stripeClient.Customers.Update(customerID, params)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	return nil
```

Comment:
You can use `return Error.Wrap(err)` with nil errors. It `Error.Wrap(nil)` returns `nil`.

Reply:
thanks!